### PR TITLE
Added racing parameters

### PR DESCRIPTION
**Pull request by joabakk**
Added definition of start line, time to start and time it takes to turn around any way to burn time before start, and layline distance. Any rounding marks are assumed to be the current destination waypoint

### CODE REVIEW

File: schemas/groups/navigation.json [hunk 1]
```diff
@@ -151,6 +151,55 @@
...
+			},
+			"distanceLayline": {
+				"type": "number",
+				"description": "The current distance to the layline",
+				"units": "m"
+			}
+			
+		}
+	}
+
```
Comment (tkurki) [inline schemas/groups/navigation.json]: Missing trailing comma after the closing curly brace

File: schemas/groups/navigation.json [hunk 2]
```diff
@@ -151,6 +151,55 @@
...
+				"units": "s"
+			},
+			"timeStbdUp": {
+				"description": "Time to arrive at the start line on starboard, turning upwind",
+				"$ref": "../definitions.json#/definitions/numberValue",
+				"units": "s"
+			},
+			"distanceLayline": {
+				"type": "number",
+				"description": "The current distance to the layline",
```
Comment (tkurki) [inline schemas/groups/navigation.json]: how about distance to startline?

**Comment by tkurki** [comment]
How about a test that asserts the vailidy of a sample file?

**Comment by joabakk** [comment]
Ok I am really sorry for all the commits, but is should be working now.

**Pull request merged**